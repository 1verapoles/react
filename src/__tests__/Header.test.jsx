import React from 'react';
import { BrowserRouter } from "react-router-dom";
import { render } from '@testing-library/react';
import Header from '../Header';

describe('Testing Header component', () => {
  it('renders without crashing', () => {
    const { asFragment } = render(<BrowserRouter><Header /></BrowserRouter>);

    expect(asFragment()).toMatchSnapshot();
  });
});
