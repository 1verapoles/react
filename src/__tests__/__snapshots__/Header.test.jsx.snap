// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Header component renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <nav
      class="navbar navbar-expand-lg navbar-light bg-light"
    >
      <button
        aria-controls="navbarNav"
        aria-expanded="false"
        aria-label="Toggle navigation"
        class="navbar-toggler"
        data-target="#navbarNav"
        data-toggle="collapse"
        type="button"
      >
        <span
          class="navbar-toggler-icon"
        />
      </button>
      <div
        class="collapse navbar-collapse"
        id="navbarNav"
      >
        <ul
          class="navbar-nav"
        >
          <li
            class="nav-item"
          >
            <a
              aria-current="page"
              class="nav-link active"
              href="/"
            >
              Home
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="/about"
            >
              About
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </div>
</DocumentFragment>
`;
